woocommerce/woocommerce-gutenberg-products-block
# Merge markup from Product Title atomic block and ProductName component

Part of woocommerce/woocommerce#42680.

Similar to woocommerce/woocommerce-blocks#3065 (that one was merging `ProductPrice` while this one is about `ProductName`). This PR replaces some of the markup of the Product Title atomic block with the ProductName component.

### How to test the changes in this Pull Request:

Testing basically means verifying there are no regressions:

1. Load the Cart and Checkout blocks. **_Verify_** product names are still rendered correctly.
![imatge](https://user-images.githubusercontent.com/3616980/102381394-6e833100-3fc9-11eb-82ef-6be403737cad.png)
2. Create a page with the Product Title block. **_Verify_** you can toggle styles in the sidebar and they are applied correctly.
![imatge](https://user-images.githubusercontent.com/3616980/102381519-a0949300-3fc9-11eb-8cac-b219f866e24f.png)
